.github/workflows/release-local-runtime.yml

```
@@ -0,0 +1,315 @@
+          fi
+
+          if [ -n "$RELEASE_TAG" ] && [[ ! "$RELEASE_TAG" =~ ^local-v.+$ ]]; then
+            echo "release_tag must match local-v<version>" >&2
+            exit 1
+          fi
+
+          if [ -n "$RELEASE_TAG" ]; then
+            RELEASE_VERSION="${RELEASE_TAG#local-v}"
+          else
+            RELEASE_VERSION="$(python3 - <<'PY'
+import json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)